src/protagonist/API/Features/Image/ImageController.cs

```diff
@@ -115,19 +117,19 @@ public async Task<IActionResult> Image(int customerId, int spaceId, string image
     }
     
     
-    // ############################# PUT IMAGE ###############################
-    // ############################# PATCH IMAGE #############################
     /// <summary>
-    /// PUT an image to its ID location.
-    /// This may be a create or an update operation.
-    /// Unlike Deliverator, the same method handles a PATCH.
+    /// PUT   /customers/{customerId}/spaces/{spaceId}/images/{imageId}
+    /// PATCH /customers/{customerId}/spaces/{spaceId}/images/{imageId}
+    /// 
+    /// PUT an asset to its ID location or PATCH asset at that location.
+    /// (Unlike Deliverator, the same method handles a PATCH.)
     /// </summary>
     /// <param name="customerId">(from resource path)</param>
     /// <param name="spaceId">(from resource path)</param>
     /// <param name="imageId">(from resource path)</param>
     /// <param name="hydraAsset">The body of the request contains the Asset in Hydra JSON-LD form (Image class)</param>
     /// <returns>The created or updated Hydra Image object for the Asset</returns>
-    [ProducesResponseType((int)HttpStatusCode.OK, Type = typeof(DLCS.HydraModel.Image))] // for Patch
+    [ProducesResponseType((int)HttpStatusCode.OK, Type = typeof(DLCS.HydraModel.Image))] // for PATCH
     [ProducesResponseType((int)HttpStatusCode.Created, Type = typeof(DLCS.HydraModel.Image))] // for PUT when created

```

Comment:
```cs
[ProducesResponseType((int)HttpStatusCode.OK, Type = typeof(DLCS.HydraModel.Image))] // for PATCH
[ProducesResponseType((int)HttpStatusCode.Created, Type = typeof(DLCS.HydraModel.Image))] // for PUT when created
```

Given that these are for specific verb types, is it worth having 2 separate action methods that internally use the same method? Keeps the fact that `PUT` + `PATCH` being handled by same logic as an implementation detail. Would also make the swagger/openapi generated docs clearer.

```cs
/// <summary>
/// PUT /customers/{customerId}/spaces/{spaceId}/images/{imageId}
///
/// PUT an asset to its ID location
/// </summary>
[ProducesResponseType((int)HttpStatusCode.OK, Type = typeof(DLCS.HydraModel.Image))]
[ProducesResponseType((int)HttpStatusCode.Created, Type = typeof(DLCS.HydraModel.Image))]
[HttpPut]
public Task<IActionResult> PutImage(..) => PutOrPatchImage(..);

/// <summary>
/// PATCH /customers/{customerId}/spaces/{spaceId}/images/{imageId}
///
/// PATCH asset at that location.
/// </summary>
[ProducesResponseType((int)HttpStatusCode.OK, Type = typeof(DLCS.HydraModel.Image))]
[HttpPatch]
public Task<IActionResult> PatchImage(..) => PutOrPatchImage(..);

private Task<IActionResult> PutOrPatchImage(..) { ... }
```